drivers/staging/android/lowmemorykiller_tng.c

```diff
@@ -302,7 +302,7 @@ void print_obituary(struct task_struct *doomed,
 		     global_node_page_state(NR_SLAB_UNRECLAIMABLE) *
 		     (long)(PAGE_SIZE / 1024),
 		     gfp_mask,
-		     global_node_page_state(NR_INDIRECTLY_RECLAIMABLE_BYTES)
+		     global_node_page_state(NR_KERNEL_MISC_RECLAIMABLE)
```

Comment:
and here this one: https://github.com/sonyxperiadev/kernel/commit/b29940c1abd7a4c3abeb926df0a5ec84d6902d47#diff-fa19cdd9c3369d7f59aa2e8404628109408dbf8e1b568d1157a27328f75b8410R681

The `		     "   Indirect Reclaimable is %zdkB\n"` above should also be renamed accordingly I believe.

Reply:
Adjusted.